docs/tutorials/tutorials-runtime-upgrade.md

```diff
@@ -49,6 +48,46 @@ sidebar_label: Runtime Upgrade
 
 * Click the _Submit proposal_ button and sign the transaction
 
+![Submit Proposal](/img/tutorial/submit_proposal.png "Submit Proposal")
+
+### Step 4: Seconding the Proposal
+
+* Select the account to second the proposal
+
+* Verify the number of tokens required to second the proposal and Click the "Second" button and sign the transaction
+
+![Second Proposal1](/img/tutorial/second_proposal1.png "Second Proposal1")
+
+![Second Proposal2](/img/tutorial/second_proposal2.png "Second Proposal2")
+
+![Second Proposal3](/img/tutorial/second_proposal3.png "Second Proposal3")
+
+### Step 5: Voting on the Proposal
+
+* The most seconded proposal moves to public referendum
+
+* Once in referendum, token holders vote "Aye" or "Nay" on the proposal by locking tokens. Two factors account the vote weight: amount locked and locking period
+
+* If the proposal passes, it is enacted after a certain amount of time
```

Comment:
a nice to have is to mention what happens if nobody votes on the referendum and it does not pass, and how they could re-submit the proposal (i.e. regenerate the same pre-image hash without submitting it, and create a new proposal)